stdlib/functools.pyi

```diff
@@ -131,3 +133,19 @@ if sys.version_info >= (3, 8):
 
 if sys.version_info >= (3, 9):
     def cache(__user_function: Callable[..., _T]) -> _lru_cache_wrapper[_T]: ...
+
+class _HashedSeq(Generic[_T]):
+    __slots__: str
+    def __init__(self, tup: Tuple[Any, ...], hash: int = ...) -> None: ...
```

Comment:
`hash` is actually a function, defaulting to the built-in `hash` function (in `builtins.pyi`):

```suggestion
    def __init__(self, tup: Tuple[Any, ...], hash: Callable[[object], int] = ...) -> None: ...
```

You will need to add `from typing import Callable` (yes, that has to be imported from `typing`).

Reply:
It is already there!

Reply:
oh! I see.